apps/io-sign-backoffice-ca/src/app/auth/_lib/session.ts

```diff
@@ -0,0 +1,75 @@
+import { cookies } from "next/headers";
+import { SignJWT, jwtVerify } from "jose";
+import { cache } from "react";
+import { z } from "zod";
+
+const defaultCookieOptions = {
+  secure: true,
+  httpOnly: true,
+  sameSite: true,
+};
+
+const cookieName = "_iosign_session";
+
+const getSecret = cache(() => {
```

Comment:
what about something like:

```suggestion
const getSecret = cache(({ secret } = process.env) => {
```

in this way you don't have to inspect the method implementation to find out which env you need to mock in tests

Reply:
this is fine as a pattern in case you need to test the functions directly, but this is not the case of `getSecret` that is tested indirectly by testing the other (exported) functions of the module